controllers/sensor/resource.go

```diff
@@ -41,7 +41,7 @@ func (ctx *sensorContext) generateServiceSpec() *corev1.Service {
 
 	serviceSpec := &corev1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        fmt.Sprintf("%s-sensor-svc", ctx.sensor.Name),
+			Name:        fmt.Sprintf("%s-svc", ctx.sensor.Name),
```

Comment:
This is the same naming convention as Gateway Service, with this, you need to ask the user never give a same name to Gateway and Sensor, otherwise there is an issue.

Reply:
I was thinking along the lines that the user will name the gateway CR as `webhook-gateway` or `calendar-gateway` as opposed to just `webhook` or `calendar`. The former is currently demonstrated through the examples. If we change all examples to the latter then it makes sense to add `-gateway` or `-sensor` to the name of the deployments and service. I am leaning towards former because if you consider any core K8s controllers like deployment or replicaset, they also don't `-deployment` or `-replicaset` in the actual pod name.

Reply:
It's not comparable to whether POD has `-deployment` or `-replicaset` in the name, POD always has a `GeneratedName`, it makes sure its name is unique. 

This is the `Name` for a Service, no matter wheter we put `-gateway` in the examples or not, users always can name both Gateway and Sensor `webhook` or a simple `test`, it leads to both Service names are `webhook-svc` or `test-svc`. We put those examples there but we didn't say "you need to strictly follow the examples to name your Gateway and Sensors".

Reply:
I'm fine with adding `-gateway` and `-sensor` to svc as long as we update all examples so that names don't become `webhook-gateway-webhook-gateway-1234` and the service like `webhook-gateway-gateway-svc`.

Reply:
Let me update the examples.

Reply:
👍